zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinQuery.thrift

```diff
@@ -78,9 +78,6 @@ struct TraceCombo {
   4: optional map<i64, i32> span_depths # not set if no spans in trace
 }
 
-/** This is obsolete. Use Order.NONE for compatibility. */
-enum Order { TIMESTAMP_DESC, TIMESTAMP_ASC, DURATION_ASC, DURATION_DESC, NONE }
```

Comment:
well, the only caller of this was web. and thrift will ignore fields it doesn't know about (ex. field 7 below).

An experiment to prove this would be to run `bin/query` and `bin/collector` from master, and then run `bin/web` from tag 1.2.2.